app/scripts/views/object/display-data.js

```diff
@@ -0,0 +1,40 @@
+import React, {PropTypes} from 'react'
+
+import i18n from '../../utils/i18n'
+import RawData from './raw-data'
+
+const DisplayData = ({data}) => {
+  const size = data && data.length - 2
+
+  if (!size) {
```

Comment:
Because you're not just asking about size, you're also checking for existence. `isData` would be better. Also, curious what the `data.length - 2` does -- what if `data == 2`? Won't that return 0, and therefore false?